WeakAurasOptions/ActionOptions.lua

```diff
@@ -716,7 +734,13 @@ function OptionsPrivate.GetActionOptions(data)
         callbacks = {
           OnEditFocusGained = function(self)
             local widget = dynamicTextInputs["finish_message"]
-            OptionsPrivate.ToggleTextReplacements(data, true, widget)
+            OptionsPrivate.ToggleTextReplacements(data, widget, "OnEditFocusGained")
+          end,
+          OnEditFocusLost = function(self)
+            OptionsPrivate.ToggleTextReplacements(nil, "OnEditFocusLost")
```

Comment:
the second argument to ToggleTextReplacement is the widget, you have multiple places where you don't pass in the right amount of parameters.